test/new-e2e/tests/agent-metrics-logs/log-agent/linux-log/integrations/fixtures/rotation.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+
+from datadog_checks.base import AgentCheck
+
+
+class HelloCheck(AgentCheck):
+    def __init__(self, name, init_config, instances):
+        super().__init__(name, init_config, instances)
+        # Initialize a local variable to simulate a counter
+        self.counter = 0
+
+    def check(self, instance):
+        self.counter = self.counter + 1
+        data = {}
+
+        uuid_str = str(uuid.uuid4())
+        # Create a 1 MB string
```

Comment:
Any reason to create just 1 1Mb file instead of say 4 256Kb? That way you wouldn't have to worry about logs being truncated. You could just loop through the check four or so times and make it configurable so can create different scenarios.

Reply:
An example of an integration that grabs some config can be found [here](https://github.com/DataDog/integrations-core/blob/d5a502f08594ff2491fd07a54e43b82b0bc18c72/argocd/datadog_checks/argocd/check.py#L41C9-L41C21).

Reply:
Overall looks good 👍🏽

Reply:
That's a good point, log truncation sort of throws a wrench into the works.

The way logs from integrations settings are configured now, `logs_config.integrations_logs_files_max_size` can only be configured in whole megabytes, though this can be changed in a future update. 

Since the agent's individual log size max is 256kb, I'm going to change the log size to 255kb, which should have the same effect of triggering a file deletion and creation every five check runs if the `logs_config.integrations_logs_files_max_size` is still set to 1mb.